docs/end2end-TrecQA-eval.md

```diff
@@ -22,8 +22,7 @@ Possible parameters are:
 -input (required)
 ```
 
-Path of a TrecQA file
-
+Path of the TrecQA test file. It can be downloaded from [here](http://cs.jhu.edu/~xuchen/packages/jacana-qa-naacl2013-data-results.tar.bz2)
```

Comment:
The link points to a compressed folder that has dev, test, and train files. We need all of them!